src/fetchcode/package.py

```diff
@@ -89,28 +88,30 @@ def get_cargo_data_from_purl(purl):
         download_url=download_url,
         **purl.to_dict(),
     )
-    versions = response.get("versions", [])
-    for version in versions:
-        version_purl = PackageURL(type=purl.type, name=name, version=version.get("num"))
-        dl_path = version.get("dl_path")
-        if dl_path:
-            download_url = f"{base_url}/{dl_path}"
-        else:
-            download_url = None
-        declared_license = version.get("license")
-
-        yield Package(
-            homepage_url=homepage_url,
-            api_url=api_url,
-            code_view_url=code_view_url,
-            download_url=download_url,
-            declared_license=declared_license,
-            **version_purl.to_dict(),
-        )
+
+    if all_versions:
```

Comment:
Please check the invert of it, this will help in removing indentation by one level.

Reply:
@TG1999 can you please tell what is the meaning of invert here and what to do.